circle.yml

```diff
@@ -1330,9 +1330,9 @@ jobs:
     steps:
       - test-binary-against-repo:
           repo: cypress-example-recipes
-          command: npm run test:ci
-          pull_request_id: 503
-          folder: examples/stubbing-spying__window-fetch
+          command: npm test
+          pull_request_id: 513
+          folder: examples/fundamentals__module-api-wrap
```

Comment:
what's this all about?

Reply:
this is for testing this branch against an example recipe that shows a nice example. Once this lands, the above branch name will be ignored, and this job won't be needed. When someone else needs to test another PR, then they can update these parameters

Reply:
can you add a comment to that effect? otherwise that "someone else" will only ever be Gleb :smile:  right now, if i saw this in `develop`, i'd assume it's junk and delete it.

Reply:
added